src/Observable.ts

```diff
@@ -3,12 +3,13 @@ import {Operator} from './Operator';
 import {Subscriber} from './Subscriber';
 import {Subscription, AnonymousSubscription, TeardownLogic} from './Subscription';
 import {root} from './util/root';
-import {$$observable} from './symbol/observable';
 import {toSubscriber} from './util/toSubscriber';
 
 import {IfObservable} from './observable/IfObservable';
 import {ErrorObservable} from './observable/ErrorObservable';
 
+import * as observableSymbol from 'symbol-observable';
```

Comment:
Nitpick: I prefer `$$name` for symbols. Can we use `$$observable`? It's consistent with the rest of the library.